README.md

```diff
@@ -48,14 +48,11 @@ ReactDOM.render(
   document.body.firstChild
 );
 ```
-**[⬆ back to top](#Contents)**

```

Comment:
Why did you remove all the links to the ToC?

Reply:
They were not working on GitHub. Will figure later why but for now they were creating noise and no value.

---
In reply to: [180925584](https://github.com/OfficeDev/office-ui-fabric-react/pull/4523#discussion_r180925584) [](ancestors = 180925584)

Reply:
I was assuming @manishgarg1 decided against them.  Looked like he added them the other day.

Reply:
No, they were just not working. So I am yanking them out. Will fix them later in a way that they are funcitonal :)

---
In reply to: [180927395](https://github.com/OfficeDev/office-ui-fabric-react/pull/4523#discussion_r180927395) [](ancestors = 180927395)